### PR TITLE
Refine scrutinizer rules to optimize speed

### PR DESCRIPTION
**Pull request by drbyte**

### CODE REVIEW

File: .scrutinizer.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            rulesets:
+                - codesize
+                - unusedcode
+                - naming
+                - design
+    sensiolabs_security_checker: true
+
+filter:
+    paths:
+        - 'includes/*'
```
Comment (texdc) [inline .scrutinizer.yml]: admin path(s) should be added

File: .scrutinizer.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    excluded_paths:
+        - 'includes/classes/vendors/*'
+        - 'includes/classes/support/*'
+        - 'includes/classes/class.phpmailer.php'
+        - 'includes/classes/class.smtp.php'
+        - 'includes/classes/http_client.php'
+        - 'includes/classes/class.phpbb.php'
+        - 'includes/library/aura/*'
+        - 'includes/modules/*'
+        - 'includes/functions/*'
```
Comment (texdc) [inline .scrutinizer.yml]: I wouldn't exclude modules or functions - mess detection can be helpful here.

File: .scrutinizer.yml (hunk 1 above)
Comment (drbyte) [inline .scrutinizer.yml]: They will be, in time.

**Comment by zcwilt** [comment]
:+1

**Comment by ajeh** [comment]
:+1:

**Pull request merged**